packages/calcite-components/src/components/list/usage/Advanced.md

```diff
@@ -1,37 +1,34 @@
 ```html
 <calcite-list>
-  <calcite-list-item label="Princess Bubblegum" description="Ruler of The Candy Kingdom">
+  <calcite-list-item
+    label="Dog"
+    description="(Canis familiaris) A carnivorous mammal closely related to the gray wolf that has been domesticated as a pet."
```

Comment:
For readability, WDYT about adding a sentence referencing the Latin name for this and other `list-item`'s? Maybe something like:

> Also known as Canis familiaris, a carnivorous...

Reply:
That makes sense to me, 👍